src/client/diagnostics.rs

```diff
@@ -0,0 +1,137 @@
+use bevy::diagnostic::DiagnosticId;
+use bevy::{
+    diagnostic::{Diagnostic, Diagnostics, RegisterDiagnostic},
+    prelude::*,
+    time::common_conditions::on_timer,
+};
+use std::time::Duration;
+
+/// Collects replication stats during packet processing
+/// flushed to Diagnostics system periodically.
```

Comment:
Official rust do usually styled like this:
```suggestion
/// Replication stats during packet processing.
///
/// Flushed to Diagnostics system periodically.
```